scripts/testing/docker-compose_otel-collector.yaml

```diff
@@ -0,0 +1,73 @@
+version: '3.9'
```

Comment:
should the files in `scripts/testing/` be in `test/integration/metrics/`?

Reply:
I'll leave these where they are for now, since there are no tests yet that use em'. As discussed at the office, I'll get to work on proper metrics integration tests in a separate PR, and I'll get these moved to `test/integration/metrics/` then.